bin/setup

```diff
@@ -20,7 +20,7 @@ chdir APP_ROOT do
 
 
   puts "\n== Copying sample files =="
-  unless File.exist?('config/database.yml')
+  unless File.exist?('db/schema.rb')
     cp 'db/schema.rb.example', 'db/schema.rb'
```

Comment:
Isn't this a shell command?

Reply:
Same for line 28

Reply:
Yes, but these methods are part of ruby standard library as well. See https://ruby-doc.org/stdlib-2.2.2/libdoc/fileutils/rdoc/FileUtils.html

Reply:
Ah, news to me! Thank you for this.